### DISCUSSION TITLE
Debugging issue with Error Code SIGLL

### DISCUSSION THREAD

**Discussion by saadh393** [vercel/next.js#69942] [Help]
### Link to the code that reproduces this issue

https://github.com/saadh393/in-code

### To Reproduce

I am using VS Code to Debug my Next.js Frontend. Whenever I try to debug with Break points it gets stuck and Chrome window gets like this -

I have also tried to debug with "debugger" statement and directly from Chrome Dev Tools. But Nothing changed.
I also removed node dependencies, .next folder and reinstalled everything but doesn't helped.

**Step by step process to reproduce the Issue.**
- Run the project with npm run dev
- Browse the page - http://localhost:3000/editor
- Open VS Code and Add some break Points with debugger statement.
- Open Chrome Devtool switch to Sources tab
- When Debugger Attached, keep step into the next function, It will crashed with Error Code SIGLL

_Note: It takes some time to crash_

### Current vs. Expected behavior

This is My Current Error. 
![image](https://github.com/user-attachments/assets/be46fd5c-f541-4270-b110-32c044252c2c)

Expected Output is to work smoothly 

### Provide environment information

```bash
Operating System:
  Platform: linux
  Arch: x64
  Version: #197-Ubuntu SMP Thu Mar 28 22:45:47 UTC 2024
  Available memory (MB): 23887
  Available CPU cores: 12
Binaries:
  Node: 18.18.0
  npm: 9.8.1
  Yarn: 1.22.19
  pnpm: 9.9.0
Relevant Packages:
  next: 14.2.8 // Latest available version is detected (14.2.8).
  eslint-config-next: 14.2.5
  react: 18.3.1
  react-dom: 18.3.1
  typescript: 5.5.4
Next.js Config:
  output: N/A
```

### Which area(s) are affected? (Select all that apply)

Not sure

### Which stage(s) are affected? (Select all that apply)

next dev (local)

**Comment by samcx** [comment]
@saadh393 I was not able to replicate this. I did notice issues with your :repro: (e.g., importing Typebox incorrectly).
